### PR TITLE
chore: update fees for Noble

### PR DESCRIPTION
**Pull request by johnletey**

### CODE REVIEW

File: cosmos/noble.json [hunk 1]
```diff
@@ -38,19 +38,21 @@
...
+        "average": 0.1,
+        "high": 0.2
       }
     },
     {
       "coinDenom": "ATOM",
       "coinMinimalDenom": "ibc/EF48E6B1A1A19F47ECAEA62F5670C37C0580E86A9E88498B7E393EB6F49F33C0",
       "coinDecimals": 6,
+      "coinGeckoId": "cosmos",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cosmoshub/uatom.png",
```
Comment (johnletey) [inline cosmos/noble.json:51]: GitHub Actions is saying that this is an [invalid image URL](https://github.com/chainapsis/keplr-chain-registry/actions/runs/7100481549/job/19326658604#step:7:9). Does anyone know why?

Comment (johnletey) [reply cosmos/noble.json:51]: @HeesungB I'm also happy to move the `uatom.png` into `images/noble` if you want!

Comment (HeesungB) [reply cosmos/noble.json:51]: @johnletey No.I think it would be nice to remove the 'coingecoid' and 'coinunknown' fields.

Comment (johnletey) [reply cosmos/noble.json]: I removed both fields in https://github.com/chainapsis/keplr-chain-registry/pull/314/commits/5d88708fa4a2b3c66ed611959fa4e714bc8f345c!

**Comment by HeesungB** [comment]
@johnletey Should I merge it?

**Comment by johnletey** [comment]
Thank you for merging @HeesungB! And please push this update out to the Keplr extension!

**Pull request merged**